core/application/commands.go

```diff
@@ -171,3 +186,24 @@ func (a *app) addInstanceData(vm *models.MicroVM, logger *logrus.Entry) error {
 
 	return nil
 }
+
+func (a *app) addMetadataInterface(mvm *models.MicroVM) error {
```

Comment:
This could probably return a `bool` rather than an `error`. The caller is doing `fmt.Errorf("adding metadata network interface: %w", err)` but the func never returns an err.
Actually, in that case does it need to return anything? It could just `return` nothing on 194 and leave the func early

Reply:
Thats a good point. Changed.